encoding.go

```diff
@@ -190,3 +186,40 @@ func (e encoder) writeLevel(buf *buffer, l slog.Level) {
 	e.writeColoredString(buf, str, style)
 	buf.AppendByte(' ')
 }
+
+func trimmedPath(path string, cwd string) string {
+	// if the file path appears to be under the current
+	// working directory, then we're probably running
+	// in a dev environment, and we can show the
+	// path of the source file relative to the
+	// working directory
+	if cwd != "" && strings.HasPrefix(path, cwd) {
+		if ff, err := filepath.Rel(cwd, path); err == nil {
+			return ff
+		}
+	}
```

Comment:
does the prefix logic match on Windows ?

Could you add a test about this ?

Thanks

Reply:
Let's continue here


https://github.com/phsym/console-slog/pull/17#issuecomment-2608101951
> Oh, also, I don't have a windows machine to generate sample values for a test.  Do you have any samples?


I don't either. When I'm unsure I tend to use GitHub CI and pop a windows image

https://github.com/actions/runner-images